### PR TITLE
OCM-9879 | test: Automated ids:66359,66372,66872,66874 for disk size

### PR DESCRIPTION
**Pull request by 86254860**

### CODE REVIEW

File: tests/e2e/hcp_machine_pool_test.go [hunk 1]
```diff
@@ -114,6 +114,50 @@ var _ = Describe("HCP Machine Pool", labels.Feature.Machinepool, func() {
...
+			Expect(textData).
+				Should(ContainSubstring(
+					"Machine pool '%s' created successfully on hosted cluster '%s'",
+					npID,
+					clusterID))
+
+			By("Check the machinepool list")
+			output, err = machinePoolService.ListMachinePool(clusterID)
+			Expect(err).ToNot(HaveOccurred())
+			defer rosaClient.MachinePool.DeleteMachinePool(clusterID, npID)
```
Comment (radtriste) [inline tests/e2e/hcp_machine_pool_test.go:140]: The delete should be on line #130, after the creation happened well

Comment (86254860) [reply tests/e2e/hcp_machine_pool_test.go:140]: Good catch! Updated.

**Review by radtriste — APPROVED** [review]
/lgtm

**Pull request merged**